godot/src/main.gd

```diff
@@ -40,7 +40,7 @@ func _start():
 		)
 	elif args.has("--scene-renderer"):
 		get_tree().change_scene_to_file("res://src/tool/scene_renderer/scene.tscn")
-	elif args.has("--scene-test") or args.has("--skip-lobby"):
+	elif args.has("--scene-test") or args.has("--skip-lobby-guest"):
```

Comment:
```suggestion
	elif args.has("--scene-test") or args.has("--skip-lobby-guest"):
```

I think we can maintain just '--skip-lobby' and always try to recover the last session; if there is not, join as a guest...